index.html

```diff
@@ -22,6 +22,11 @@
                 background-color: red;
             }
 
+            button.page-link:focus {
+                box-shadow: none;
+                outline: 2px solid black;
```

Comment:
For reference the output of this css looks like this:
![image](https://github.com/iSlammedMyKindle/lemonbotHomepage/assets/108752129/8224f644-ea2b-4386-945f-22e1a647842f)

I'm not sure I like a black & bold highlight, it's purely personal preference, but my thought is that it kindof clashes with everything else on the page, as nothing else has borders except for this collection of bootstrap elements.

The element itself having borders is alright I guess (that's a bootstrap thing), but having them get even bigger doesn't feel like it works with the page well, but that's purely opinion.

I would personally take it off unless there's another reason to have it, that will be the main reason why I mark the PR as request changes

Reply:
I know it looks ugly, but I added the border because of accessibility reasons. If you remove it it's impossible to know that element is on focus and that makes using the website more difficult for people who rely on keyboard controls. It could be changed though

Reply:
I changed the outline css so it uses the default outline provided by the browser, it looks better. Without it it uses Bootstrap's outline which looks even worse (it has a blue glow)

Reply:
![image](https://github.com/iSlammedMyKindle/lemonbotHomepage/assets/108752129/732f0954-06f8-450b-986a-888327c51502)
This is what shows on microsoft edge, looks nice!

![image](https://github.com/iSlammedMyKindle/lemonbotHomepage/assets/108752129/cafbd0c1-f59b-43fd-b0a2-2bbfc53f3b54)

On firefox there doesn't seem to be an outline at all, but I wonder if it's because of my theme

Reply:
I fixed it so it looks better on Firefox now. Thanks for noticing!